include/swift/Frontend/Frontend.h

```diff
@@ -436,6 +420,23 @@ class CompilerInstance {
   /// \brief Returns true if there was an error during setup.
   bool setup(const CompilerInvocation &Invocation);
 
+private:
+  void setUpLLVMArguments();
+  void setUpDiagnosticOptions();
+  bool setUpModuleLoaders();
+  Optional<unsigned> setUpCodeCompletionBuffer();
+  bool setupInputs(Optional<unsigned> codeCompletionBufferID);
```

Comment:
Still wrong capitalization for `setupInputs`, `setupForInput`, and `setupForBuffer`.

Reply:
Argh! I *know* I fixed those somewhere. Probably in a future PR. But I will fix them here, too. Sorry.